src/WebUI/Types.d.ts

```diff
@@ -25,6 +26,8 @@ export interface IDeploymentItem {
     statusProps?: IStatusProps;
     showRowBorder?: boolean;
     deployment?: V1Deployment;
+    image: string;

```

Comment:
is it mandatory to have an image for deployment? if not, make this as image?: string